docs/guide/repository.md

```diff
@@ -746,14 +598,14 @@ Persistence cache uses a separate Cache Pool decorator which by design prefixes
 /** @var $cacheService \eZ\Publish\Core\Persistence\Cache\CacheServiceDecorator */
 $cacheService = $container->get('ezpublish.cache_pool.spi.cache.decorator');
```

Comment:
these four lines can be removed (wrong variable + type, and getting pool is described above)

ref:
```
// getting the cache service in php
 
  /** @var $cacheService \eZ\Publish\Core\Persistence\Cache\CacheServiceDecorator */
  $cacheService = $container->get('ezpublish.cache_pool.spi.cache.decorator');
```